prefect_gcp/cloud_storage.py

```diff
@@ -1140,3 +1141,77 @@ async def upload_from_folder(
         if num_uploaded == 0:
             self.logger.warning(f"No files were uploaded from {from_folder}.")
         return bucket_folder
+
+    @sync_compatible
+    async def upload_from_dataframe(
+        self,
+        df: pd.DataFrame,
+        to_path: str,
+        output_format: str = "parquet",
+        compression: Optional[str] = None,
```

Comment:
I think we might want to use a Literal type for both these keywords, e.g. `output_format: Literal["parquet", "csv"] = "csv"`

Reply:
Fixed it! 👍🏻